wolox-books/src/scss/_equalizer.scss

```diff
@@ -0,0 +1,92 @@
+$equalizer-black: #000;
```

Comment:
Actually, this file is no longer needed. You can use our own equalizer https://www.npmjs.com/package/wolox-equalizer 😄

Reply:
Sure! Package added and implemented!